eve/workers/pod-builder/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM centos:7
+
+COPY requirements-eve.txt /tmp/
```

Comment:
In general it's better to move 'things that change' after 'things that are less likely to change'. As such, I'd suggest:
```
RUN yum-config-manager --add-repo ... && \
    yum install -y gcc make ... && \
    adduser ... &&
    (no real need for `usermod`, just set things up as part of `adduser`)

ADD requirements-eve.txt /tmp
RUN pip install -r /tmp/... && pip install ...
```

Also, I'm quite surprised why on one end there's a `requirements.txt` for a set of 'fairly random' Python packages, but then there's a separate install for `buildbot-worker`. Why not keep that in `requirements.txt` as well?

Reply:
frankly I dont know, I can do that I just find most of eve dockerfile workers on this format

Reply:
Ping @tcarmet

Reply:
I also appreciate leaving everything in `requirements.txt` as long as you set a version for the buildbot-worker. Could be any version as long as it's newer than the one you have as a build arg.